examples/remix-auth-email-link/app/routes/logout.tsx

```diff
@@ -0,0 +1,16 @@
+import type { ActionFunction, LoaderFunction } from 'remix'
+import { json, redirect } from 'remix'
+
+import { destroySession, getSession } from '~/services/session.server'
+
+export const action: ActionFunction = async ({ request }) => {
```

Comment:
Could you add comments to explain why it's important that this is an `action` with a `Form` instead of using `loader` with a `Link`? (CSRF for example).